specs/linux/ipmi_lan.table

```diff
@@ -0,0 +1,19 @@
+table_name("ipmi_lan")
+description("IPMI LAN information.")
+schema([
+    Column("mc_name", TEXT, "Name of the MC"),
+    Column("mc_id", TEXT, "Device ID of the MC"),
```

Comment:
hmmm... I guess it's a preference thing then.  I've always thought that it's a note clear for a more general attribute name like (name, id, etc) to be prefixed with another description if they're not actually belonging to the item of that table.  For instance, if `ipmi_threshold_sensors` had a `id` attribute, I'd naturally think `id` meant that each sensor had it's own `id`, and not that it's the `id` of the MC that it belongs to.  I hope that made sense haha

Right now, you would join with something like this:
```sql
select * from ipmi_threshold_sensors join ipmi_mc on ipmi_mc.id = ipmi_threshold_sensors.mc_id;
```
Thoughts?